servers/extensions/physics_server_3d_extension.h

```diff
@@ -271,8 +271,11 @@ class PhysicsServer3DExtension : public PhysicsServer3D {
 	EXBIND2RC(Variant, area_get_param, RID, AreaParameter)
 	EXBIND1RC(Transform3D, area_get_transform, RID)
 
-	EXBIND2(area_set_collision_mask, RID, uint32_t)
 	EXBIND2(area_set_collision_layer, RID, uint32_t)
+	EXBIND1RC(uint32_t, area_get_collision_layer, RID)
+	
```

Comment:
Please delete the extraneous tab character at the beginning of the (otherwise empty) line.